src/templates/next/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,17 @@
+import { NotificationProps } from "~/common/Notification"
+import BaseParagraph from "../shared/Paragraph"
+import { RxCross2 } from "react-icons/rx"
+
+const Notification = ({ title }: NotificationProps) => {
+  return (
+    <div className="flex flex-col sm:flex-row justify-between gap-6 bg-[#FEF4E8] py-4 px-5 sm:px-10">
```

Comment:
nit: I think the `#fef4e8` here represents `site.primary.100`, so we can use that colour token instead (can refer to `next.ts` inside the `presets` folder for the colour tokens specific to the next theme).